src/lib/zf/zf_stackdump.c

```diff
@@ -26,6 +26,41 @@ static void dump_zockets(SkewPointer<zf_stack> stack, Zocket* zockets,
   }
 }
 
+void dump_attributes(SkewPointer<zf_stack_impl> stimpl)
+{
+  zf_dump("---------------------attributes-------------------------------\n");
+  if(stimpl->sti_tx_ring_max==-1){
+  zf_dump("tx_ring size= 512\n");
```

Comment:
nit: indentation should be increased inside the if statement, also on line 39 (below here)

Reply:
For single if/else branches the curly brackets are not needed '{' and to match coding styles there should be spaces in-between the condition and brackets. For example 
`if( condition )
  function();
else
  other_function();`